libshpool/src/daemon/server.rs

```diff
@@ -197,6 +197,7 @@ impl Server {
 
         let (child_exit_notifier, inner_to_stream, pager_ctl_slot, status) = {
             // we unwrap to propagate the poison as an unwind
+            let _s = span!(Level::INFO, "1_lock(shells)").entered();
```

Comment:
for reviewing all these, i suggest we have a naming convention.

im assuming here you have chosen one: the convention `<supplemental_details>_lock(<lock_name>)`

do we have one and what is it (if we agree to have one)?

Reply:
Yeah that's the convention I choose